jetstream_pt/engine.py

```diff
@@ -507,8 +508,10 @@ def _load_from_safetensors(self, path):
         arr = jax.device_put(f.get_tensor(key), self.sharding_by_name(key))
         assert tuple(model_weights.shape) == tuple(arr.shape), f"key: {key} error: {model_weights.shape} != {arr.shape}"
         weights[key] = arr
+        print(f"key:{key}, value shape:{arr.dtype}")
```

Comment:
nit: Please follow the same pattern as in 529, and only print layer 0 for better user experience.

Reply:
Thanks, updated.